src/service/FlywheelService.ts

```diff
@@ -1,3 +1,4 @@
+import { Flowdsl } from '@flowfact/node-flowdsl/lib/Flowdsl';
```

Comment:
```suggestion
import { Flowdsl } from '@flowfact/node-flowdsl/lib/Flowdsl';
```
Siehe Zeile darunter